idaes/generic_models/properties/core/examples/tests/test_CO2_H2O_Ideal_VLE.py

```diff
@@ -0,0 +1,390 @@
+        sv = model.props[1].define_state_vars()
+
+        assert len(sv) == 4
+        for i in sv:
+            assert i in ["flow_mol",
+                         "mole_frac_comp",
+                         "temperature",
+                         "pressure"]
+
+    @pytest.mark.unit
+    def test_define_display_vars(self, model):
+        sv = model.props[1].define_display_vars()
+
+        assert len(sv) == 4
+        for i in sv:
+            assert i in ["Total Molar Flowrate",
+                         "Total Mole Fraction",
+                         "Temperature",
+                         "Pressure"]
+
+    @pytest.mark.unit
+    def test_dof(self, model):
+        # Fix state
+        model.props[1].flow_mol.fix(10)
+        model.props[1].temperature.fix(323.15)
+        model.props[1].pressure.fix(108900)
+        model.props[1].mole_frac_comp["H2O"].fix(0.5)
+        model.props[1].mole_frac_comp["CO2"].fix(0.5)
+
+        assert degrees_of_freedom(model.props[1]) == 0
+
+    @pytest.mark.initialize
+    @pytest.mark.solver
+    @pytest.mark.skipif(solver is None, reason="Solver not available")
+    @pytest.mark.component
+    def test_initialize(self, model):
+        orig_fixed_vars = fixed_variables_set(model)
+        orig_act_consts = activated_constraints_set(model)
+
+        model.props.initialize(optarg={'tol': 1e-6})
+
+        assert degrees_of_freedom(model) == 0
+
+        fin_fixed_vars = fixed_variables_set(model)
+        fin_act_consts = activated_constraints_set(model)
+
+        assert len(fin_act_consts) == len(orig_act_consts)
+        assert len(fin_fixed_vars) == len(orig_fixed_vars)
+
+        for c in fin_act_consts:
+            assert c in orig_act_consts
+        for v in fin_fixed_vars:
+            assert v in orig_fixed_vars
+
+    @pytest.mark.solver
+    @pytest.mark.skipif(solver is None, reason="Solver not available")
+    @pytest.mark.component
+    def test_solve(self, model):
+        results = solver.solve(model)
+
+        # Check for optimal solution
+        assert results.solver.termination_condition == \
+            TerminationCondition.optimal
+        assert results.solver.status == SolverStatus.ok
+
+    @pytest.mark.initialize
+    @pytest.mark.solver
+    @pytest.mark.skipif(solver is None, reason="Solver not available")
+    @pytest.mark.component
+    def test_solution(self, model):
+        # Check phase equilibrium results
+        assert model.props[1].mole_frac_phase_comp["Liq", "H2O"].value == \
+            pytest.approx(1, abs=1e-4)
+        assert model.props[1].mole_frac_phase_comp["Vap", "H2O"].value == \
+            pytest.approx(0.11501, abs=1e-4)
+        assert model.props[1].mole_frac_phase_comp["Vap", "CO2"].value == \
+            pytest.approx(0.88499, abs=1e-4)
+        assert model.props[1].phase_frac["Vap"].value == \
+            pytest.approx(0.56498, abs=1e-4)
+            
+    @pytest.mark.integration
+    def test_temp_swing(self,model):
```

Comment:
You can take the `model` argument out here, as you are not actually using the fixture.

Reply:
Yes, i will make this change. Thanks.